### PR TITLE
[TT-5450] update swagger

### PR DESCRIPTION
**Pull request by jeffy-mathew**
## Description
update swagger.yml to group OAS endpoints.
correct OAS patch endpoint.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Refactoring or add test (improvements in base code or adds test coverage to functionality)

## Checklist

- [ ] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). If pulling from your own
      fork, don't request your `master`!
- [ ] Make sure you are making a pull request against the **`master` branch** (left side). Also, you should start
      *your branch* off *our latest `master`*.
- [ ] My change requires a change to the documentation.
  - [ ] If you've changed APIs, describe what needs to be updated in the documentation.
  - [ ] If new config option added, ensure that it can be set via ENV variable
- [ ] I have updated the documentation accordingly.
- [ ] Modules and vendor dependencies have been updated; run `go mod tidy && go mod vendor`
- [ ] When updating library version must provide reason/explanation for this update.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] Check your code additions will not fail linting checks:
  - [ ] `go fmt -s`
  - [ ] `go vet`

### CODE REVIEW

File: swagger.yml [hunk 1]
```diff
@@ -315,15 +315,15 @@ paths:
...
+        - OASAPIs
       operationId: listApisOAS
       parameters:
-      - description: mode of export
-        name: mode
-        in: query
-        schema:
-          type: string
-        example: public
+        - description: mode of export
```
Comment (lghiur) [inline swagger.yml:321]: can we extend a bit on this, explaining what are the mdodes?

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/pull/4069/merge
Commit:  
Triggered by: pull_request (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2377896337)

File: swagger.yml (hunk 1 above)
Comment (jeffy-mathew) [reply swagger.yml]: done

**Review by lghiur — APPROVED** [review]

**Review by furkansenharputlu — APPROVED** [review]

**Comment by Tyk-ITS** [comment]
**API tests result: success** :white_check_mark: 
Branch used: refs/heads/master
Commit: 1f35d42584df76549f8aa9191fc68e48fe53051a [TT-5450] update swagger (#4069)

* update swagger.
correct PATCH endpoint
group oas endpoints

[changelog]
internal: update swagger.yml to group OAS endpoints.
internal: correct OAS patch endpoint.
Triggered by: push (@jeffy-mathew)
[Execution page](https://github.com/TykTechnologies/tyk/actions/runs/2378198951)

**Pull request merged**